cmd/warm-backend-s3.go

```diff
@@ -128,10 +128,9 @@ func newWarmBackendS3(conf madmin.TierS3) (*warmBackendS3, error) {
 	if err != nil {
 		return nil, err
 	}
-	core, err := minio.NewCore(u.Host, opts)
-	if err != nil {
-		return nil, err
-	}
+	client.SetAppInfo("s3-tier-target", ReleaseTag)
```

Comment:
Can we go one step further and differentiate among traffic to different storage tiers?
```suggestion
	client.SetAppInfo(fmt.Sprintf("s3-tier-target-%s", conf.StorageClass), ReleaseTag)
```